geode-wan/src/test/java/org/apache/geode/cache/wan/WANRollingUpgradeDUnitTest.java

```diff
@@ -548,19 +551,36 @@ public void testEventProcessingMixedSiteOneCurrentSiteTwo() throws Exception {
     rollStartAndConfigureServerToCurrent(site1Server2, site1Locators, site2DistributedSystemId,
         regionName, site1SenderId, ParallelGatewaySenderQueue.DEFAULT_MESSAGE_SYNC_INTERVAL);
 
-    // Start and configure old site servers
+    // Start and configure old current servers
     String site2SenderId = getName() + "_gatewaysender_" + site1DistributedSystemId;
     startAndConfigureServers(site2Server1, site2Server2, site2Locators, site1DistributedSystemId,
         regionName, site2SenderId, ParallelGatewaySenderQueue.DEFAULT_MESSAGE_SYNC_INTERVAL);
 
-    // Do puts from mixed site client and verify events on old site
+    // Do puts from mixed site client and verify events on current site
     int numPuts = 100;
     doClientPutsAndVerifyEvents(site1Client, site1Server1, site1Server2, site2Server1, site2Server2,
         hostName, site1LocatorPort, regionName, numPuts, site1SenderId, false);
   }
 
   private void startLocator(int port, int distributedSystemId, String locators,
       String remoteLocators) throws IOException {
+    Properties props = getLocatorProperties(distributedSystemId, locators, remoteLocators);
+    Locator.startLocatorAndDS(port, null, props);
+  }
+
+  private int startLocatorWithJmxManager(int port, int distributedSystemId, String locators,
+      String remoteLocators) throws IOException {
+    Properties props = getLocatorProperties(distributedSystemId, locators, remoteLocators);
+    int jmxPort = AvailablePortHelper.getRandomAvailableTCPPort();
+    props.put(DistributionConfig.JMX_MANAGER_PORT_NAME, String.valueOf(jmxPort));
```

Comment:
Could we possibly change all the references to the `DistributionConfig` and replace it with `ConfigurationProperties`

We should not be referencing any config properties from `DistributionConfig` any more...

Reply:
Done. Thanks.